app.py

```diff
@@ -54,6 +54,34 @@ def duplicate_dataset(ds_name, new_ds_name):
         res = "Sorry, something went wrong in our dataset duplication. Contact the admin for more information."
     return res
 
+@app.route('/duplicate-datasets', methods=['GET', 'POST'])
+def duplicate_datasets():
+    """
+    Duplicate a list of datasets
+
+    body: A list of datasets to be duplicated in the format:
+    [
+        {
+            "ds_name": "dataset1",
+            "new_ds_name": "new_dataset1"
+        },
+        {
+            "ds_name": "dataset2",
+            "new_ds_name": "new_dataset2"
+        }
+    ]
+    """
+    data = request.get_json()
+    logging.debug(f"route: /duplicate-datasets - Duplicating dataset {len(data)} datasets")  # noqa: E501
+    try:
+        for ds in data:
+            duplicate_ssr_parsed_files(ds['ds_name'], ds['new_ds_name'])
```

Comment:
this try catch block will only catch if there is no ds_name or new_ds_name provided. The child function which processes the error can possibly return "Sorry, something went wrong in our SSR duplication. Contact the admin for more information."

Maybe check: 
```
res = duplicate_ss.....
if res != "Success":
  raise ValueError(res)
```

Reply:
Yeah I think I should so a count for the successful datasets.